app/plugins/cookies.js

```diff
@@ -8,11 +8,17 @@ module.exports = {
       server.state('cookies_policy', config)
 
       server.ext('onPreResponse', (request, h) => {
+        const pageUrl = request?.headers?.referer?.split('/').pop()
+        if (!validSession(request) && pageUrl !== 'start' && pageUrl !== 'session-timeout') {
```

Comment:
Should probably not show the timeout page on the `cookies` or `accessibility` page either